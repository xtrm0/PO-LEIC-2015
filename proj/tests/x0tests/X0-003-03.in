5
Hello,
Is there anybody in there?
4
5
Just nod if you can hear me.
Is there anyone at home?
4
5
Come on now
I hear you're feeling down
4
5
Well, I can ease your pain
And get you on your feet again
4
5
Relax
I'll need some information first
4
5
Just the basic facts
Can you show me where it hurts?
4
5
There is no pain, you are receding
A distant ship smoke on the horizon
4
5
You are only coming through in waves
Your lips move but I can't hear what you're saying
4
5
When I was a child I had a fever
My hands felt just like two balloons
4
5
Now I've got that feeling once again
I can't explain, you would not understand
4
5
This is not how I am
I have become comfortably numb
4
5
I have become comfortably numb
O.K.
4
5
Just a little pin prick
There'll be no more aaaaaaaah!
4
5
But you may feel a little sick
Can you stand up?
4
5
I do believe it's working, good
That'll keep you going through the show
4
5
Come on, it's time to go.
There is no pain you are receding
4
5
A distant ship smoke on the horizon
You are only coming through in waves
4
5
Your lips move but I can't hear what you're saying
When I was a child
4
5
I caught a fleeting glimpse
Out of the corner of my eye
4
5
I turned to look but it was gone
I cannot put my finger on it now
4
5
The child is grown
The dream is gone
4
5
I have become comfortably numb.
¯\_(ツ)_/¯
4
0
0
0
