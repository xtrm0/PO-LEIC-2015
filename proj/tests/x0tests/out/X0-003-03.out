Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: Hello,/Is there anybody in there?
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: Hello,/Is there anybody in there?
Autor: Just nod if you can hear me./Is there anyone at home?
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: Come on now/I hear you're feeling down
Autor: Hello,/Is there anybody in there?
Autor: Just nod if you can hear me./Is there anyone at home?
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: Come on now/I hear you're feeling down
Autor: Hello,/Is there anybody in there?
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Well, I can ease your pain/And get you on your feet again
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: Come on now/I hear you're feeling down
Autor: Hello,/Is there anybody in there?
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Relax/I'll need some information first
Autor: Well, I can ease your pain/And get you on your feet again
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: Come on now/I hear you're feeling down
Autor: Hello,/Is there anybody in there?
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Relax/I'll need some information first
Autor: Well, I can ease your pain/And get you on your feet again
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: Come on now/I hear you're feeling down
Autor: Hello,/Is there anybody in there?
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Relax/I'll need some information first
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: Well, I can ease your pain/And get you on your feet again
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: Come on now/I hear you're feeling down
Autor: Hello,/Is there anybody in there?
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Relax/I'll need some information first
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: Well, I can ease your pain/And get you on your feet again
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: Come on now/I hear you're feeling down
Autor: Hello,/Is there anybody in there?
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Relax/I'll need some information first
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: Well, I can ease your pain/And get you on your feet again
Autor: When I was a child I had a fever/My hands felt just like two balloons
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: Come on now/I hear you're feeling down
Autor: Hello,/Is there anybody in there?
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Now I've got that feeling once again/I can't explain, you would not understand
Autor: Relax/I'll need some information first
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: Well, I can ease your pain/And get you on your feet again
Autor: When I was a child I had a fever/My hands felt just like two balloons
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: Come on now/I hear you're feeling down
Autor: Hello,/Is there anybody in there?
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Now I've got that feeling once again/I can't explain, you would not understand
Autor: Relax/I'll need some information first
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: This is not how I am/I have become comfortably numb
Autor: Well, I can ease your pain/And get you on your feet again
Autor: When I was a child I had a fever/My hands felt just like two balloons
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: Come on now/I hear you're feeling down
Autor: Hello,/Is there anybody in there?
Autor: I have become comfortably numb/O.K.
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Now I've got that feeling once again/I can't explain, you would not understand
Autor: Relax/I'll need some information first
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: This is not how I am/I have become comfortably numb
Autor: Well, I can ease your pain/And get you on your feet again
Autor: When I was a child I had a fever/My hands felt just like two balloons
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: Come on now/I hear you're feeling down
Autor: Hello,/Is there anybody in there?
Autor: I have become comfortably numb/O.K.
Autor: Just a little pin prick/There'll be no more aaaaaaaah!
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Now I've got that feeling once again/I can't explain, you would not understand
Autor: Relax/I'll need some information first
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: This is not how I am/I have become comfortably numb
Autor: Well, I can ease your pain/And get you on your feet again
Autor: When I was a child I had a fever/My hands felt just like two balloons
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: But you may feel a little sick/Can you stand up?
Autor: Come on now/I hear you're feeling down
Autor: Hello,/Is there anybody in there?
Autor: I have become comfortably numb/O.K.
Autor: Just a little pin prick/There'll be no more aaaaaaaah!
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Now I've got that feeling once again/I can't explain, you would not understand
Autor: Relax/I'll need some information first
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: This is not how I am/I have become comfortably numb
Autor: Well, I can ease your pain/And get you on your feet again
Autor: When I was a child I had a fever/My hands felt just like two balloons
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: But you may feel a little sick/Can you stand up?
Autor: Come on now/I hear you're feeling down
Autor: Hello,/Is there anybody in there?
Autor: I do believe it's working, good/That'll keep you going through the show
Autor: I have become comfortably numb/O.K.
Autor: Just a little pin prick/There'll be no more aaaaaaaah!
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Now I've got that feeling once again/I can't explain, you would not understand
Autor: Relax/I'll need some information first
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: This is not how I am/I have become comfortably numb
Autor: Well, I can ease your pain/And get you on your feet again
Autor: When I was a child I had a fever/My hands felt just like two balloons
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: But you may feel a little sick/Can you stand up?
Autor: Come on now/I hear you're feeling down
Autor: Come on, it's time to go./There is no pain you are receding
Autor: Hello,/Is there anybody in there?
Autor: I do believe it's working, good/That'll keep you going through the show
Autor: I have become comfortably numb/O.K.
Autor: Just a little pin prick/There'll be no more aaaaaaaah!
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Now I've got that feeling once again/I can't explain, you would not understand
Autor: Relax/I'll need some information first
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: This is not how I am/I have become comfortably numb
Autor: Well, I can ease your pain/And get you on your feet again
Autor: When I was a child I had a fever/My hands felt just like two balloons
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: A distant ship smoke on the horizon/You are only coming through in waves
Autor: But you may feel a little sick/Can you stand up?
Autor: Come on now/I hear you're feeling down
Autor: Come on, it's time to go./There is no pain you are receding
Autor: Hello,/Is there anybody in there?
Autor: I do believe it's working, good/That'll keep you going through the show
Autor: I have become comfortably numb/O.K.
Autor: Just a little pin prick/There'll be no more aaaaaaaah!
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Now I've got that feeling once again/I can't explain, you would not understand
Autor: Relax/I'll need some information first
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: This is not how I am/I have become comfortably numb
Autor: Well, I can ease your pain/And get you on your feet again
Autor: When I was a child I had a fever/My hands felt just like two balloons
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: A distant ship smoke on the horizon/You are only coming through in waves
Autor: But you may feel a little sick/Can you stand up?
Autor: Come on now/I hear you're feeling down
Autor: Come on, it's time to go./There is no pain you are receding
Autor: Hello,/Is there anybody in there?
Autor: I do believe it's working, good/That'll keep you going through the show
Autor: I have become comfortably numb/O.K.
Autor: Just a little pin prick/There'll be no more aaaaaaaah!
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Now I've got that feeling once again/I can't explain, you would not understand
Autor: Relax/I'll need some information first
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: This is not how I am/I have become comfortably numb
Autor: Well, I can ease your pain/And get you on your feet again
Autor: When I was a child I had a fever/My hands felt just like two balloons
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Autor: Your lips move but I can't hear what you're saying/When I was a child
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: A distant ship smoke on the horizon/You are only coming through in waves
Autor: But you may feel a little sick/Can you stand up?
Autor: Come on now/I hear you're feeling down
Autor: Come on, it's time to go./There is no pain you are receding
Autor: Hello,/Is there anybody in there?
Autor: I caught a fleeting glimpse/Out of the corner of my eye
Autor: I do believe it's working, good/That'll keep you going through the show
Autor: I have become comfortably numb/O.K.
Autor: Just a little pin prick/There'll be no more aaaaaaaah!
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Now I've got that feeling once again/I can't explain, you would not understand
Autor: Relax/I'll need some information first
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: This is not how I am/I have become comfortably numb
Autor: Well, I can ease your pain/And get you on your feet again
Autor: When I was a child I had a fever/My hands felt just like two balloons
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Autor: Your lips move but I can't hear what you're saying/When I was a child
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: A distant ship smoke on the horizon/You are only coming through in waves
Autor: But you may feel a little sick/Can you stand up?
Autor: Come on now/I hear you're feeling down
Autor: Come on, it's time to go./There is no pain you are receding
Autor: Hello,/Is there anybody in there?
Autor: I caught a fleeting glimpse/Out of the corner of my eye
Autor: I do believe it's working, good/That'll keep you going through the show
Autor: I have become comfortably numb/O.K.
Autor: I turned to look but it was gone/I cannot put my finger on it now
Autor: Just a little pin prick/There'll be no more aaaaaaaah!
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Now I've got that feeling once again/I can't explain, you would not understand
Autor: Relax/I'll need some information first
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: This is not how I am/I have become comfortably numb
Autor: Well, I can ease your pain/And get you on your feet again
Autor: When I was a child I had a fever/My hands felt just like two balloons
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Autor: Your lips move but I can't hear what you're saying/When I was a child
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: A distant ship smoke on the horizon/You are only coming through in waves
Autor: But you may feel a little sick/Can you stand up?
Autor: Come on now/I hear you're feeling down
Autor: Come on, it's time to go./There is no pain you are receding
Autor: Hello,/Is there anybody in there?
Autor: I caught a fleeting glimpse/Out of the corner of my eye
Autor: I do believe it's working, good/That'll keep you going through the show
Autor: I have become comfortably numb/O.K.
Autor: I turned to look but it was gone/I cannot put my finger on it now
Autor: Just a little pin prick/There'll be no more aaaaaaaah!
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Now I've got that feeling once again/I can't explain, you would not understand
Autor: Relax/I'll need some information first
Autor: The child is grown/The dream is gone
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: This is not how I am/I have become comfortably numb
Autor: Well, I can ease your pain/And get you on your feet again
Autor: When I was a child I had a fever/My hands felt just like two balloons
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Autor: Your lips move but I can't hear what you're saying/When I was a child
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: 
Autor: A distant ship smoke on the horizon/You are only coming through in waves
Autor: But you may feel a little sick/Can you stand up?
Autor: Come on now/I hear you're feeling down
Autor: Come on, it's time to go./There is no pain you are receding
Autor: Hello,/Is there anybody in there?
Autor: I caught a fleeting glimpse/Out of the corner of my eye
Autor: I do believe it's working, good/That'll keep you going through the show
Autor: I have become comfortably numb/O.K.
Autor: I have become comfortably numb./¯\_(ツ)_/¯
Autor: I turned to look but it was gone/I cannot put my finger on it now
Autor: Just a little pin prick/There'll be no more aaaaaaaah!
Autor: Just nod if you can hear me./Is there anyone at home?
Autor: Just the basic facts/Can you show me where it hurts?
Autor: Now I've got that feeling once again/I can't explain, you would not understand
Autor: Relax/I'll need some information first
Autor: The child is grown/The dream is gone
Autor: There is no pain, you are receding/A distant ship smoke on the horizon
Autor: This is not how I am/I have become comfortably numb
Autor: Well, I can ease your pain/And get you on your feet again
Autor: When I was a child I had a fever/My hands felt just like two balloons
Autor: You are only coming through in waves/Your lips move but I can't hear what you're saying
Autor: Your lips move but I can't hear what you're saying/When I was a child
Secções de topo: 0
Dimensão do documento (bytes): 0
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: 